### ISSUE TITLE
Depends on non-existent module

### ISSUE THREAD

**Issue by eserte** [seagirl/dwarf#16]
While trying to build Cmd-Dwarf-1.27 using CPAN.pm:

```
Running test for module 'Data::Path'

  The module Data::Path isn't available on CPAN.

  Either the module has not yet been uploaded to CPAN, or it is
  temporary unavailable. Please contact the author to find out
  more about the status. Try 'i Data::Path'.
```

Also, Data::Path is not available on MetaCPAN. https://metacpan.org/pod/Data::Path returns a 404 page.

**Comment by seagirl** [comment]
Hi, eserte

Thank you for reporting.

I found Bug Ticket related this

Data::Path is missing in CPAN index
https://rt.cpan.org/Public/Bug/Display.html?id=55483

you can install like below

```
% cpan -i ZAPHAR/Data-Path-1.4.1.tar.gz
```

or use cpanm

```
% cpanm Data::Path
```

http://search.cpan.org/~zaphar/Data-Path-1.4.1/

**Comment by eserte** [comment]
And it was even me who reported the problem, but this was 5 years (!) ago. So it basically looks like Data::Path is unmaintained. Maybe there are alternatives (e.g. Data::DPath)? Or maybe, as you're a user of Data::Path, you want to take over maintainership of the module?

**Comment by seagirl** [comment]
Thank you for your support!

Now I released Cmd-Dwarf-1.28 and it uses Data::DPath instead of Data::Path.
http://search.cpan.org/~seagirl/Cmd-Dwarf-1.28/

**Comment by eserte** [comment]
Looks fine now!
